### PR TITLE
Resolves #2303

### LINKED ISSUE #2303: App-VpnConnection with Custom EAP Configuration XML Example
Another helpful example for Add-VpnConnection would be using a custom EAP configuration XML such as produced by following the process here https://docs.microsoft.com/en-us/windows/client-management/mdm/eap-configuration.

Example 4 shows using New-EapConfiguration to generate a configuration, however that cmdlet doesn't support the full set of available configurations.

We kept trying to modify a EAP configuration variable created by New-EapConfiguration following Example 4, when all we had to do was take the existing XML string we had in a variable and cast it as XML then use it in Add-VpnConnection.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 52b71d06-6ad9-81ed-f75a-aec0b7737306
* Version Independent ID: 338bfafe-80bd-1bdb-8d9b-d43a04e007c6
* Content: [Add-VpnConnection (VpnClient)](https://docs.microsoft.com/en-us/powershell/module/vpnclient/add-vpnconnection?view=win10-ps)
* Content Source: [docset/windows/vpnclient/Add-VpnConnection.md](https://github.com/MicrosoftDocs/windows-powershell-docs/blob/master/docset/windows/vpnclient/Add-VpnConnection.md)
* Product: **w10**
* Technology: ****
* GitHub Login: @Kateyanne
* Microsoft Alias: **v-kaunu**

### PR DESCRIPTION
**Pull request by velkovb**
Add Example
https://github.com/MicrosoftDocs/windows-powershell-docs/issues/2303

### CODE REVIEW

File: docset/windows/vpnclient/Add-VpnConnection.md [hunk 1]
```diff
@@ -121,27 +127,21 @@ SplitTunneling        : True
 ```
 
 This command adds a VPN connection named Test4 to the server with an IP address of 10.1.1.1.
-This connection uses the default EAP authentication method, as specified by the *AuthenticationMethod* parameter.
-The pre-shared key for the connection is specified by the *L2tpPsk* parameter Additional parameters specify that the connection: 
+This connection uses the default EAP authentication method, as specified by the **AuthenticationMethod** parameter.
+The pre-shared key for the connection is specified by the **L2tpPsk** parameter Additional parameters specify that the connection:
```
Comment (JohanFreelancer9) [inline docset/windows/vpnclient/Add-VpnConnection.md:131]:
```suggestion
The pre-shared key for the connection is specified by the **L2tpPsk** parameter. Additional parameters specify that the connection: 
```

File: docset/windows/vpnclient/Add-VpnConnection.md [hunk 2]
```diff
@@ -159,7 +159,15 @@ SplitTunneling        : True
...
-For more information about custom EAP authentication methods, see the New-EapConfiguration cmdlet.
+For more information about custom EAP authentication methods, see the **New-EapConfiguration** cmdlet.
+
+### Example 5: Add a VPN connection that uses already generated EAP XML configuration
+```powershell
+PS C:\> $EAPXml = [xml]Get-Content -Path C:\eap-config.xml
+PS C:\> Add-VpnConnection -Name "Test6" -ServerAddress "10.1.1.1" -TunnelType "L2tp" -EncryptionLevel "Required" -AuthenticationMethod Eap -SplitTunneling -AllUserConnection -RememberCredential -EapConfigXmlSteam $EAPXml
+```
+
+This set of commands adds a VPN connection using already generated EAP XML configuration. For more information how to generate EAP XML configuration, see [EAP Configuration](https://docs.microsoft.com/windows/client-management/mdm/eap-configuration)
```
Comment (JohanFreelancer9) [inline docset/windows/vpnclient/Add-VpnConnection.md:170]:
```suggestion
This set of commands adds a VPN connection using an already generated EAP XML configuration. For more information about how to generate EAP XML configuration, see [EAP Configuration](https://docs.microsoft.com/windows/client-management/mdm/eap-configuration)
```

File: docset/winserver2012-ps/vpnclient/Add-VpnConnection.md [hunk 3]
```diff
@@ -104,28 +108,20 @@ SplitTunneling        : True
 
 This command adds a VPN connection named Test4 to the server with an IP address of 10.1.1.1.
 This connection uses the default EAP authentication method, as specified by the **AuthenticationMethod** parameter.
-The pre-shared key for the connection, password, is specified by the **L2tpPsk** parameter Additional parameters specify that the connection: 
+The pre-shared key for the connection is specified by the **L2tpPsk** parameter Additional parameters specify that the connection:
```
Comment (JohanFreelancer9) [inline docset/winserver2012-ps/vpnclient/Add-VpnConnection.md:111]:
```suggestion
The pre-shared key for the connection is specified by the **L2tpPsk** parameter. Additional parameters specify that the connection: 
```

File: docset/winserver2012-ps/vpnclient/Add-VpnConnection.md [hunk 4]
```diff
@@ -143,7 +139,15 @@ SplitTunneling        : True
...
-For more information about custom EAP authentication methods, see the New-EapConfiguration cmdlet.
+For more information about custom EAP authentication methods, see the **New-EapConfiguration** cmdlet.
+
+### Example 5: Add a VPN connection that uses already generated EAP XML configuration
+```powershell
+PS C:\> $EAPXml = [xml]Get-Content -Path C:\eap-config.xml
+PS C:\> Add-VpnConnection -Name "Test6" -ServerAddress "10.1.1.1" -TunnelType "L2tp" -EncryptionLevel "Required" -AuthenticationMethod Eap -SplitTunneling -AllUserConnection -RememberCredential -EapConfigXmlSteam $EAPXml
+```
+
+This set of commands adds a VPN connection using already generated EAP XML configuration. For more information how to generate EAP XML configuration, see [EAP Configuration](https://docs.microsoft.com/windows/client-management/mdm/eap-configuration)
```
Comment (JohanFreelancer9) [inline docset/winserver2012-ps/vpnclient/Add-VpnConnection.md:150]:
```suggestion
This set of commands adds a VPN connection using an already generated EAP XML configuration. For more information about how to generate EAP XML configuration, see [EAP Configuration](https://docs.microsoft.com/windows/client-management/mdm/eap-configuration)
```

File: docset/winserver2012r2-ps/vpnclient/Add-VpnConnection.md [hunk 5]
```diff
@@ -123,23 +129,17 @@ This connection uses the default EAP authentication method, as specified by the
 The pre-shared key for the connection is specified by the **L2tpPsk** parameter Additional parameters specify that the connection:
```
Comment (JohanFreelancer9) [inline docset/winserver2012r2-ps/vpnclient/Add-VpnConnection.md:129]:
```suggestion
The pre-shared key for the connection is specified by the **L2tpPsk** parameter. Additional parameters specify that the connection: 
```

File: docset/winserver2012r2-ps/vpnclient/Add-VpnConnection.md [hunk 6]
```diff
@@ -157,7 +157,15 @@ SplitTunneling        : True
...
-For more information about custom EAP authentication methods, see the New-EapConfiguration cmdlet.
+For more information about custom EAP authentication methods, see the **New-EapConfiguration** cmdlet.
+
+### Example 5: Add a VPN connection that uses already generated EAP XML configuration
+```powershell
+PS C:\> $EAPXml = [xml]Get-Content -Path C:\eap-config.xml
+PS C:\> Add-VpnConnection -Name "Test6" -ServerAddress "10.1.1.1" -TunnelType "L2tp" -EncryptionLevel "Required" -AuthenticationMethod Eap -SplitTunneling -AllUserConnection -RememberCredential -EapConfigXmlSteam $EAPXml
+```
+
+This set of commands adds a VPN connection using already generated EAP XML configuration. For more information how to generate EAP XML configuration, see [EAP Configuration](https://docs.microsoft.com/windows/client-management/mdm/eap-configuration)
```
Comment (JohanFreelancer9) [inline docset/winserver2012r2-ps/vpnclient/Add-VpnConnection.md:168]:
```suggestion
This set of commands adds a VPN connection using already generated EAP XML configuration. For more information about how to generate EAP XML configuration, see [EAP Configuration](https://docs.microsoft.com/windows/client-management/mdm/eap-configuration).
```

File: docset/winserver2019-ps/vpnclient/Add-VpnConnection.md [hunk 7]
```diff
@@ -121,27 +127,21 @@ SplitTunneling        : True
 ```
 
 This command adds a VPN connection named Test4 to the server with an IP address of 10.1.1.1.
-This connection uses the default EAP authentication method, as specified by the *AuthenticationMethod* parameter.
-The pre-shared key for the connection is specified by the *L2tpPsk* parameter Additional parameters specify that the connection: 
+This connection uses the default EAP authentication method, as specified by the **AuthenticationMethod** parameter.
+The pre-shared key for the connection is specified by the **L2tpPsk** parameter Additional parameters specify that the connection:
```
Comment (JohanFreelancer9) [inline docset/winserver2019-ps/vpnclient/Add-VpnConnection.md:131]:
```suggestion
The pre-shared key for the connection is specified by the **L2tpPsk** parameter. Additional parameters specify that the connection: 
```

File: docset/winserver2019-ps/vpnclient/Add-VpnConnection.md [hunk 8]
```diff
@@ -159,7 +159,15 @@ SplitTunneling        : True
...
-For more information about custom EAP authentication methods, see the New-EapConfiguration cmdlet.
+For more information about custom EAP authentication methods, see the **New-EapConfiguration** cmdlet.
+
+### Example 5: Add a VPN connection that uses already generated EAP XML configuration
+```powershell
+PS C:\> $EAPXml = [xml]Get-Content -Path C:\eap-config.xml
+PS C:\> Add-VpnConnection -Name "Test6" -ServerAddress "10.1.1.1" -TunnelType "L2tp" -EncryptionLevel "Required" -AuthenticationMethod Eap -SplitTunneling -AllUserConnection -RememberCredential -EapConfigXmlSteam $EAPXml
+```
+
+This set of commands adds a VPN connection using already generated EAP XML configuration. For more information how to generate EAP XML configuration, see [EAP Configuration](https://docs.microsoft.com/windows/client-management/mdm/eap-configuration)
```
Comment (JohanFreelancer9) [inline docset/winserver2019-ps/vpnclient/Add-VpnConnection.md:170]:
```suggestion
This set of commands adds a VPN connection using already generated EAP XML configuration. For more information about how to generate EAP XML configuration, see [EAP Configuration](https://docs.microsoft.com/windows/client-management/mdm/eap-configuration).
```

File: docset/winserver2012r2-ps/vpnclient/Add-VpnConnection.md [hunk 9]
```diff
@@ -554,7 +559,7 @@ Accept wildcard characters: False
 ```
 
 ### CommonParameters
-This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see about_CommonParameters (http://go.microsoft.com/fwlink/?LinkID=113216).
+This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](https://go.microsoft.com/fwlink/?LinkID=113216).
```
Comment (JohanFreelancer9) [inline docset/winserver2012r2-ps/vpnclient/Add-VpnConnection.md:562]:
```suggestion
This cmdlet supports the common parameters: `-Debug`, `-ErrorAction`, `-ErrorVariable`,
`-InformationAction`, `-InformationVariable`, `-OutVariable`, `-OutBuffer`, `-PipelineVariable`,
`-Verbose`, `-WarningAction`, and `-WarningVariable`. For more information, see [about_CommonParameters](https://go.microsoft.com/fwlink/?LinkID=113216).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by tseward — APPROVED** [review]

File: docset/windows/vpnclient/Add-VpnConnection.md [hunk 10]
```diff
@@ -159,7 +159,15 @@ SplitTunneling        : True
 ```
 
 This set of commands adds a VPN connection using a custom EAP authentication method.
-For more information about custom EAP authentication methods, see the New-EapConfiguration cmdlet.
+For more information about custom EAP authentication methods, see the **New-EapConfiguration** cmdlet.
+
+### Example 5: Add a VPN connection that uses already generated EAP XML configuration
+```powershell
+PS C:\> $EAPXml = [xml]Get-Content -Path C:\eap-config.xml
+PS C:\> Add-VpnConnection -Name "Test6" -ServerAddress "10.1.1.1" -TunnelType "L2tp" -EncryptionLevel "Required" -AuthenticationMethod Eap -SplitTunneling -AllUserConnection -RememberCredential -EapConfigXmlSteam $EAPXml
```
Comment (get-itips) [inline docset/windows/vpnclient/Add-VpnConnection.md:167]:
```suggestion
PS C:\> Add-VpnConnection -Name "Test6" -ServerAddress "10.1.1.1" -TunnelType "L2tp" -EncryptionLevel "Required" -AuthenticationMethod Eap -SplitTunneling -AllUserConnection -RememberCredential -EapConfigXmlStream $EAPXml
```

Comment (get-itips) [reply docset/windows/vpnclient/Add-VpnConnection.md:167]: Small typo, please kindly fix the other occurrences.

File: docset/windows/vpnclient/Add-VpnConnection.md [hunk 11]
```diff
@@ -159,7 +159,15 @@ SplitTunneling        : True
...
+```
+
+This set of commands adds a VPN connection using already generated EAP XML configuration. For more information about how to generate EAP XML configuration, see [EAP Configuration](https://docs.microsoft.com/windows/client-management/mdm/eap-configuration).
```
Comment (dariomws) [inline docset/windows/vpnclient/Add-VpnConnection.md:170]:
```suggestion
This set of commands adds a VPN connection using an already generated EAP XML configuration. For more information about how to generate EAP XML configuration, see [EAP Configuration](https://docs.microsoft.com/windows/client-management/mdm/eap-configuration).
```
Just a small suggestion. I approve after @get-itips and @JohanFreelancer9 details.

**Review by dariomws — APPROVED** [review]

**Review by get-itips — APPROVED** [review]

**Pull request merged**